pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/bindings/LTensorTNCudaBindings.hpp

```diff
@@ -47,6 +48,9 @@ namespace py = pybind11;
 namespace Pennylane::LightningTensor::TNCuda {
 using TensorNetBackends =
     Pennylane::Util::TypeList<MPSTNCuda<float>, MPSTNCuda<double>, void>;
+using AllTensorNetBackends =
```

Comment:
Are `TensorNetBackends` actually used anywhere? can we call this one with MPS and ExactTN `TensorNetBackends`? If the other one is also used, can we call the other one `MPSTensorNetBackends` instead?

Reply:
You are right, `TensorNetBackends` is no longer used. I will remove it